extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeMetaInfoHelper.java

```diff
@@ -171,15 +171,40 @@ private static void getEmergencyOneboxRenderer(
             // usually an encouragement like "We are with you"
             final String title = getTextFromObjectOrThrow(r.getObject("title"), "title");
             // usually a phone number
-            final String action = getTextFromObjectOrThrow(r.getObject("actionText"), "action");
+            final String action;
+            if (r.has("actionText")) {
+                action = getTextFromObjectOrThrow(r.getObject("actionText"), "action");
+            } else if (r.has("contacts")) {
+                final JsonArray contacts = r.getArray("contacts");
+                final StringBuilder stringBuilder = new StringBuilder();
+                int i = 0;
+                final int contactsSize = contacts.size();
+                if (contactsSize != 0) {
+                    // Loop over contacts item from the first contact to the last one, if there is
+                    // not only one, in order to not add an unneeded line return
+                    for (; i < contactsSize - 1; i++) {
+                        stringBuilder.append(getTextFromObjectOrThrow(contacts.getObject(i)
+                                .getObject("actionText"), "contacts.actionText"));
+                        stringBuilder.append("\n");
+                    }
+                    // Add the latest contact without an extra line return
+                    stringBuilder.append(getTextFromObjectOrThrow(contacts.getObject(i)
+                            .getObject("actionText"), "contacts.actionText"));
+                }
+
+                action = stringBuilder.toString();
+            } else {
+                action = null;
+            }
             // usually details about the phone number
             final String details = getTextFromObjectOrThrow(r.getObject("detailsText"), "details");
             // usually the name of an association
             final String urlText = getTextFromObjectOrThrow(r.getObject("navigationText"),
                     "urlText");
 
             metaInfo.setTitle(title);
-            metaInfo.setContent(new Description(details + "\n" + action, Description.PLAIN_TEXT));
+            metaInfo.setContent(new Description(isNullOrEmpty(action)
+                    ? details : details + "\n" + action, Description.PLAIN_TEXT));
```

Comment:
Followup to the previous suggestion
```suggestion
            metaInfo.setContent(new Description(details + action, Description.PLAIN_TEXT));
```

Reply:
I will so append for the previous/legacy structure to `details` a line return.

Reply:
What do you mean?